pkgs/by-name/lo/lovely-injector/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+let
+  version = "0.5.0-unstable-2024-07-20";
+  lovelyInjector = fetchFromGitHub {
+    # Use this fork for the PR in progress (PR #66) (currently pointing 0.5.0.beta-5)
+    # https://github.com/ethangreen-dev/lovely-injector/pull/66
+    owner = "vgskye";
+    repo = "lovely-injector";
+    rev = "a5ce6c5188ae71b3908fa3188e3a1b29c565fb65";
+    hash = "sha256-UgUgM+HJ0jrRWqIubHcNQNMwDV/ftlpZxyrIn1bs7bI=";
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "lovely-injector";
+  inherit version;
+  src = lovelyInjector;
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "retour-0.4.0-alpha.2" = "sha256-GtLTjErXJIYXQaOFLfMgXb8N+oyHNXGTBD0UeyvbjrA=";
+    };
+  };
+  # Disable tests, they are broken on my machine
+  doCheck = false;
+  # lovely-injector depends on nightly features
+  RUSTC_BOOTSTRAP = 1;
```

Comment:
```suggestion
  env.RUSTC_BOOTSTRAP = 1;
```

Reply:
I applied the `env.RUSTC_BOOTSTRAP` trick but enabling the tests still results in a crash on my system. Lovely still works but the tests do not pass.

I will open an issue and refer to this MR.